mentoring/tests/integration/xml/assessment_1.xml

```diff
@@ -14,32 +14,32 @@
 
     <pb-answer name="goal" question="What is your goal?" />
 
-    <pb-mcq name="mcq_1_1" question="Do you like this MCQ?" correct_choices="yes">
+    <pb-mcq name="mcq_1_1" question="Do you like this MCQ?" correct_choices='["yes"]'>
```

Comment:
This change seems like it may break existing problems - is it backwards compatible, or does this now have to be in a list?